app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,56 @@
+class TodosController < ApplicationController
```

Comment:
A `before_action` could be used to reduce the number of times you call `Todo.find(params[:id]`.